src/screens/Episode/components/Player/Player.tsx

```diff
@@ -0,0 +1,120 @@
+} from 'src/screens/Episode/common'
+import { getTime } from 'src/screens/Episode/helpers'
+import { PlainText } from 'src/components'
+import PlayIcon from 'src/assets/images/play.svg'
+import PauseIcon from 'src/assets/images/pause.svg'
+import RewindIcon from 'src/assets/images/rewind.svg'
+import ForwardIcon from 'src/assets/images/forward.svg'
+import styles from './styles'
+
+type Props = {
+  episode: PlayerEpisode
+}
+
+const Player: React.FC<Props> = ({ episode }) => {
+  const { position, duration } = useProgress()
+  const [isPlaying, setPlaying] = useState(false)
+
+  const trackPlayerInit = async () => {
+    await TrackPlayer.setupPlayer({
+      maxCacheSize: 1048576
+    })
+    return true
+  }
+
+  useEffect(() => {
+    trackPlayerInit()
+    TrackPlayer.updateOptions({
+      stopWithApp: false,
+      alwaysPauseOnInterruption: true,
+      capabilities: [Capability.Play, Capability.Pause, Capability.Stop],
+      compactCapabilities: [Capability.Play, Capability.Pause, Capability.Stop],
+      notificationCapabilities: [
+        Capability.Play,
+        Capability.Pause,
+        Capability.Stop
+      ]
+    })
+    TrackPlayer.add(episode)
+  }, [])
+
+  const onPressFunction = () => {
+    TrackPlayer.play()
+    setPlaying(true)
+  }
+  const onPressPause = () => {
+    TrackPlayer.pause()
+    setPlaying(false)
+  }
+  const onHandleSeekTo = (seconds: number) => {
+    TrackPlayer.seekTo(seconds)
+  }
+
+  return (
+    <View>
+      <View style={styles.sliderWrapper}>
+        <View style={styles.positionWrapper}>
+          <PlainText label={getTime(position)} style={styles.time} />
+        </View>
+        <Slider
+          style={styles.slider}
+          minimumValue={DEFAULT_START_TIME}
+          maximumValue={duration}
+          minimumTrackTintColor="#52527a"
+          maximumTrackTintColor="#52527a"
+          thumbTintColor="#52527a"
+          value={position}
+          onSlidingComplete={(seek) => onHandleSeekTo(seek)}
+        />
+        <View style={styles.durationWrapper}>
+          <PlainText label={getTime(duration)} />
+        </View>
+      </View>
+      <View style={styles.row}>
+        <Pressable
+          onPress={() => onHandleSeekTo(position - TIME_SHIFT_IN_SECONDS)}
+          style={styles.jumpButton}
+        >
+          <RewindIcon
+            style={styles.jumpIcon}
+            width={26}
+            height={26}
+            fill="#595959"
+          />
+        </Pressable>
+
+        {!isPlaying && (
+          <Pressable onPress={onPressFunction} style={styles.controlButton}>
+            <PlayIcon width={12} fill="#595959" />
+          </Pressable>
+        )}
+        {isPlaying && (
+          <Pressable onPress={onPressPause} style={styles.controlButton}>
+            <PauseIcon width={12} height={12} fill="#595959" />
+          </Pressable>
+        )}
+
+        <Pressable
+          onPress={() => onHandleSeekTo(position + TIME_SHIFT_IN_SECONDS)}
```

Comment:
This is the same to previous onPress function. Let's move it to separate function. And, please, avoid inline functions